auth/services/oauth/relying_party.go

```diff
@@ -188,115 +128,6 @@ func (s *relyingParty) RequestRFC003AccessToken(ctx context.Context, jwtGrantTok
 	return accessTokenResponse, nil
 }
 
-func (s *relyingParty) RequestRFC021AccessToken(ctx context.Context, requester did.DID, verifier did.DID, scopes string) (*oauth.TokenResponse, error) {
-	iamClient := iam.NewHTTPClient(s.strictMode, s.httpClientTimeout, s.httpClientTLS)
-	metadata, err := iamClient.OAuthAuthorizationServerMetadata(ctx, verifier)
-	if err != nil {
-		return nil, fmt.Errorf("failed to retrieve remote OAuth Authorization Server metadata: %w", err)
-	}
-
-	// get the presentation definition from the verifier
-	presentationDefinition, err := s.presentationDefinition(ctx, metadata.PresentationDefinitionEndpoint, scopes)
-	if err != nil {
-		return nil, fmt.Errorf("failed to retrieve presentation definition: %w", err)
-	}
-
-	walletCredentials, err := s.wallet.List(ctx, requester)
-	if err != nil {
-		return nil, fmt.Errorf("failed to retrieve wallet credentials: %w", err)
-	}
-
-	// match against the wallet's credentials
-	// if there's a match, create a VP and call the token endpoint
-	// If the token endpoint succeeds, return the access token
-	// If no presentation definition matches, return a 412 "no matching credentials" error
-	builder := presentationDefinition.PresentationSubmissionBuilder()
-	builder.AddWallet(requester, walletCredentials)
-
-	// Find supported VP format, matching support from:
-	// - what the local Nuts node supports
-	// - the presentation definition "claimed format designation" (optional)
-	// - the verifier's metadata (optional)
-	formatCandidates := credential.OpenIDSupportedFormats(oauth.DefaultOpenIDSupportedFormats())
-	if metadata.VPFormats != nil {
-		formatCandidates = formatCandidates.Match(credential.OpenIDSupportedFormats(metadata.VPFormats))
-	}
-	if presentationDefinition.Format != nil {
-		formatCandidates = formatCandidates.Match(credential.DIFClaimFormats(*presentationDefinition.Format))
-	}
-	format := pe.ChooseVPFormat(formatCandidates.Map)
-	if format == "" {
-		return nil, errors.New("requester, verifier (authorization server metadata) and presentation definition don't share a supported VP format")
-	}
-	// TODO: format parameters (alg, proof_type, etc.) are ignored, but should be used in the actual signing
```

Comment:
disappeared from `BuildSubmission`, has this been fixed?

Reply:
no added todo at the right location.